rewrite-core/src/main/java/org/openrewrite/config/RecipeDescriptor.java

```diff
@@ -63,5 +63,5 @@ public class RecipeDescriptor {
 
     List<RecipeExample> examples;
 
-    URI source;
+    RecipeOrigin origin;
```

Comment:
With the new manifest approach is there any reason to logically group license and source location together? Or can we just add a new field here for the license?

Reply:
But keep in mind, we should support License Name and URL to support custom (i.e. User Defined) licenses as 
well.